src/OpenTK.Mathematics/MathHelper.cs

```diff
@@ -189,23 +189,20 @@ public static float InverseSqrtFast(float x)
         /// which is found in the Quake III source code. This implementation comes from
         /// http://www.codemaestro.com/reviews/review00000105.html. For the history of this method, see
         /// http://www.beyond3d.com/content/articles/8/.
+        /// double magic number from: https://cs.uwaterloo.ca/~m32rober/rsqrt.pdf
+        /// chapter 4.8.
         /// </remarks>
         public static double InverseSqrtFast(double x)
```

Comment:
fixed this one as well, because it was a pretty easy and fast one. And I think it would still fit in here, because everything was in .Math.